index.html

```diff
@@ -1456,95 +1453,60 @@ <h3 id="navigation-scope-security-considerations">
+          the <a>steps for post-processing the <code>scope</code> member</a>
+          with <var>manifest</var>, <var>manifest URL</var>, <var>document
+          URL</var>, <var>start URL</var> as arguments.
           </li>
-          <li>Let <var>prefer related applications</var> of <var>parsed
-          manifest</var> be the result of running the <a>steps for processing
-          the <code>prefer_related_applications</code> member</a> with
-          <var>manifest</var> as argument.
+          <li>Let <var>manifest["<a>theme_color</a>"]</var> be the result of
+          running the <a>steps for post-processing the <code>theme_color</code>
+          member</a> with <var>manifest</var> as argument.
           </li>
-          <li>Let <var>theme color</var> of <var>parsed manifest</var> be the
-          result of running the <a>steps for processing the
-          <code>theme_color</code> member</a> with <var>manifest</var> as
-          argument.
-          </li>
-          <li>Let <var>background_color</var> of <var>parsed manifest</var> be
-          the result of running the <a>steps for processing the
+          <li>Let <var>manifest["<a>background_color</a>"]</var> be the result
+          of running the <a>steps for post-processing the
           <code>background_color</code> member</a> with <var>manifest</var> as
           argument.
           </li>
-          <li>Let <var>categories</var> of <var>parsed manifest</var> be the
-          result of running the <a>steps for processing the
-          <code>categories</code> member</a> with <var>manifest</var> as
-          argument.
+          <li>Let <var>manifest["<a>categories</a>"]</var> be the result of
+          running the <a>steps for post-processing the <code>categories</code>
+          member</a> with <var>manifest</var> as argument.
           </li>
-          <li>Let <var>iarc_rating_id</var> of <var>parsed manifest</var> be
-          the result of running the <a>steps for processing the
-          <code>iarc_rating_id</code> member</a> with <var>manifest</var> as
-          argument.
+          <li>Let <var>manifest["<a>icons</a>"]</var> be the result of running
+          the <a>steps for post-processing ImageResource members</a> with <var>
+            manifest</var>, <var>manifest URL</var>, and "icons" as arguments.
           </li>
-          <li>Let <var>screenshots</var> of <var>parsed manifest</var> be the
-          result of running the <a>steps for processing an array of images</a>
+          <li>Let <var>manifest["<a>screenshots</a>"]</var> be the result of
+          running the <a>steps for post-processing ImageResource members</a>
           with <var>manifest</var>, <var>manifest URL</var>, and "screenshots"
           as arguments.
           </li>
-          <li>Return <var>parsed manifest</var>.
+          <li>Let <var>manifest["<a>related_applications</a>"]</var> be the
+          result of running the <a>steps for processing the
```

Comment:
Why are this, and the next, ones called "processing" rather than "post-processing"?

Reply:
fixed